draft-ietf-quic-transport.md

```diff
@@ -1163,6 +1163,11 @@ SHOULD ensure that the pool of connection IDs available to its peer allows the
 peer to use a new connection ID on migration, as the peer will be unable to
 respond if the pool is exhausted.
 
+An endpoint that selects a zero-length connection ID during the handshake
+cannot issue a new connection ID.  A zero-length Destination Connection ID
+field is used in all packets sent toward such an endpoint over any network
+path.
```

Comment:
Sorry I know this has been merged already but I thought this is easier than open a new issue. 

How can you use a connection with zero-length Conn ID over different paths? Does an address change without Conn ID mean that the connections breaks?

Reply:
The peer could still change its address, right?

Reply:
If you have changed the address and don't have a conn ID, you cannot associate the packet to an existing connection anymore, no?

Reply:
I see your point. You can still associate it in theory, though it might be fragile - this is left to the implementation ultimately. I agree with you that the "over any network path" is mostly redundant, but I am not sure that it hurts because I don't think it suggests something that is impossible.